### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by jak103**
Small update tot he license so that we can try out a pull request.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -20,3 +20,5 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
+Added a line to the license statement.
```
Comment (bryhansen) [inline LICENSE]: Lovely addition to the code.

Comment (jak103) [inline LICENSE]: I just felt like we really needed this line here.

Comment (bryhansen) [inline LICENSE]: Yes?

**Comment by jak103** [comment]
I approve. You should accept and merge.

**Pull request merged**